### DISCUSSION TITLE
How to set more than 1 replicas for the operator

### DISCUSSION THREAD

**Discussion by joseacl** [strimzi/strimzi-kafka-operator#7301] [Q&A] [answered]
Hi,
We wanted to use the new feature coming with the new strimzi release, regarding using more than 1 replica for the operator, but I can see that the helm chart has hardcoded the number of replicas to 1, making it no possible to indicate more than one.
Is this expected?

https://github.com/strimzi/strimzi-kafka-operator/blob/main/helm-charts/helm3/strimzi-kafka-operator/templates/060-Deployment-strimzi-cluster-operator.yaml#L12

Thanks a lot in advance!

**Comment by scholzj** [comment, marked as answer]
Maybe it needs to be extended? TBH, I don't really use the Helm Charts, so I do not know how they work. With normal installation from YAML files, it also has only one replica by default. But you can basically just scale the operator with `kubectl scale ...` afterwards.

**Reply by joseacl** [reply]
I would say so.
In our case, manual scaling is not an option since we're using Argo cd to deploy it.

Thanks for the response.

**Reply by scholzj** [reply]
I opened #7302 for it.
